src/tools.rs

```diff
@@ -708,6 +708,132 @@ pub(crate) fn inc_and_check<T: PrimInt + AddAssign + std::fmt::Debug>(
     Ok(())
 }
 
+/// Spawns a named asynchronous task if the `tokio_unstable` feature is enabled.
+///
+/// Spawns a new asynchronous task, returning a [tokio::task::JoinHandle] for it.
+/// The provided future will start running in the background immediately when spawn is called, even if you don't await the returned JoinHandle.
+/// See [tokio::task::spawn].
+///
+/// If the rustflag `tokio_unstable` is active, the task will be given the specified `name`
+/// for easier identification in monitoring tools (like tokio-console).
+/// If `tokio_unstable` is not set, the task is spawned normally without a name.
+///
+/// # Parameters
+///
+/// - `name`: The name of the task (used only if `tokio_unstable` is enabled).
```

Comment:
```suggestion
/// - `name`: The name of the task.
```
It's already explained right above, so may be omitted